### DISCUSSION TITLE
Option to change the html tag generated

### DISCUSSION THREAD

**Discussion by kleine-b-avt** [justintaddei/v-wave#280] [Ideas]
It would be nice to have this option, in cases when we have to use last-of-type css selectors.

**Comment by justintaddei** [comment]
That's an excellent suggestion! I'll create an issue from this discussion to track this feature. I have a minor version feature release planned for next week that I can work this into.

**Comment by justintaddei** [comment]
@kleine-b-avt this has been implemented as the [`tagName` option](https://github.com/justintaddei/v-wave#tagname) and was just released as part of [v1.4.0](https://www.npmjs.com/package/v-wave/v/1.4.0)!

**Reply by kleine-b-avt** [reply]
@justintaddei, cool! thanks for your time mate

**Reply by Awhux** [reply]
thx for release mate ;)
